whipper/command/cd.py

```diff
@@ -525,7 +525,12 @@ def ripIfNotRipped(number):
                 track.indexes[1].relative = 0
                 continue
 
-            ripIfNotRipped(i + 1)
+            try:
+                ripIfNotRipped(i + 1)
+            except Exception as e:
```

Comment:
In general it's considered bad form to catch *all* exception classes with Exception in python, especially if they're not re-raised. (What if a KeyboardInterrupt was generated during ripIfNotRipped?) I know morituri has done this in at least one place, but that doesn't mean we want to introduce more.

Consider figuring out which specific exception you want to catch (perhaps the RuntimeError?), or modify ripIfNotRipped to either raise a semantic exception class or return a value indicating success or failure instead.